frontend/src/components/samples/SamplesListContent.js

```diff
@@ -16,15 +16,20 @@ import api, {withToken}  from "../../utils/api"
 import {listTable, setFilter, setFilterOption, clearFilters, setSortBy} from "../../modules/samples/actions";
 import {actionDropdown} from "../../utils/templateActions";
 import {prefillTemplatesToButtonDropdown} from "../../utils/prefillTemplates";
-import {withContainer, withIndividual} from "../../utils/withItem";
+import WITH_ITEM from "../../utils/withItem";
 import {SAMPLE_FILTERS} from "../filters/descriptions";
 import getFilterProps from "../filters/getFilterProps";
 import getNFilters from "../filters/getNFilters";
 import FiltersWarning from "../filters/FiltersWarning";
 import SamplesFilters from "./SamplesFilters";
 import mergedListQueryParams from "../../utils/mergedListQueryParams";
+import { WithItemComponent } from "../shared/WithItemComponent"
 
-const getTableColumns = (containersByID, individualsByID, projectsByID, sampleKinds) => [
+const getTableColumns = (containersByID, individualsByID, projectsByID, sampleKinds) => {
+  const withContainer = WithItemComponent(WITH_ITEM.withContainer)
```

Comment:
Lets call these functions 'withContainerComponent' 'withSampleComponent', etc.. to avoid getting them confused with the existing 'withContainer' or 'withSample' functions. Otherwise, searching across the project for every usage of 'withSample' gets confusing and difficult.

It will also make it easier to find any components that haven't been updated yet.